src/style.css

```
@@ -0,0 +1,30 @@
+body {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  /* justify-content: center;
+  align-items: center; */
+}
+
+.no_active {
+  background-image: url("images/star.svg");
+}
+
+.active {
+  background-image: url("images/star-active.svg");
+}
```

The block with stars--N modifier should have exactly N first stars active, according to [requirements](https://github.com/mate-academy/layout_stars#the-task)

Please try to make it work with `nth-child` pseudoclass

```suggestion
.stars--1 .stars__star:nth-child(-n+1), {
  ... more selectors to target every line of stars ... 
  background-image: url("./images/star-active.svg");
}
```